### ISSUE TITLE
Controller not showing when you open video in a new tab

### ISSUE THREAD

**Issue by adachoinw** [codebicycle/videospeed#135]
Pressing V doesn't work.
It only shows when you click on the next video on the right side playlist.

**Comment by h-h-h-h** [comment]
The controls are also not showing up for me over YouTube videos at first. Only after navigating to another video.

It might be relevant in this context, that YouTube uses the `pushState()` API.

**Comment by scorpion421** [comment]
Duplicate of #127

**Comment by codebicycle** [comment]
The new version 0.6.3.3 should fix the controller not showing in Firefox 87.0
